### PR TITLE
Add routing

### PR DESCRIPTION
**Pull request by PavelTitov16**
Add routing with working examples using few Chakra UI library styles for application
1.
image
2.
image
3.
image

### CODE REVIEW

File: src/Pages/Categories.jsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+          <h2>Welcome to the categories!</h2>
+        </main>
+        <nav>
+          <Link to='/'>Home</Link>
+        </nav>
+        </Flex>
+      </Box>
+      </>
+    );
+}
```
Comment (lazzzaroni) [inline src/Pages/Categories.jsx:19]: For some reason pre-commit hook with code formatting doesn't work on your machine.
Please run `npm run format` and commit the result.

Also try to update all dependencies by removing "node_modules" folder and running `npm i` again.

Comment (PavelTitov16) [reply src/Pages/Categories.jsx:19]: forgot, sorry :) fixed

**Comment by itsnotpolza** [comment]
did you just delete icons?

**Comment by itsnotpolza** [comment]
@PavelTitov16 add screenshots to the pull request description :)

**Comment by PavelTitov16** [comment]
> @PavelTitov16 add screenshots to the pull request description :)

did it :)

**Comment by lazzzaroni** [comment]
> > Also we can safely delete default svg's from assets folder - we don't need them in the future.

Not really, there is a reference to favicon in index.html and i would love for the icon to be in the head actually. But we can do that in a separate pull request
 

Thanks! I'll be more careful next time!

**Review by itsnotpolza — APPROVED** [review]

**Review by lazzzaroni — APPROVED** [review]
Looks like we can merge this PR!

**Pull request merged**